### PR TITLE
Crud mysql

### PR DESCRIPTION
**Pull request by medinam6**
Please review my CRUD in files server/index.js, controller/index.js, and model/index.js. Thank you

### CODE REVIEW

File: controller/index.js [hunk 1]
```diff
@@ -21,13 +21,68 @@ function getRating(shoeId) {
...
+      reject(err);
+    });
+  }
+)};
+
+function deleteReview(id) {
+  return new Promise((resolve, reject) => {
+    model.deleteReview(id)
+    .then(num => {
+      if (num == 1) {
```
Comment (jvanboch) [inline controller/index.js]: should this be a strict equals (===)?

File: server/index.js [hunk 2]
```diff
@@ -29,4 +35,46 @@ app.get('/shoes/:shoeId/rating', (req, res) => {
...
+    res.end();
+  });
+});
+
+//delete review
+app.delete('/shoes/reviews/:id', (req, res) => {
+  let { id } = req.params;
+  controller.deleteReview(id)
+  .then(data => {
+    res.send(data);
```
Comment (jvanboch) [inline server/index.js:58]: What data are you sending on a delete? Maybe send the status code for a successful delete?

File: server/index.js [hunk 3]
```diff
@@ -29,4 +35,46 @@ app.get('/shoes/:shoeId/rating', (req, res) => {
...
+  });
+});
+
+//update review
+app.put('/shoes/reviews/:id', (req, res) => {
+  let { id } = req.params;
+  let updatedReview = req.body;
+  controller.updateReview(id, updatedReview)
+  .then(data => {
+    res.send(data);
```
Comment (jvanboch) [inline server/index.js:72]: What does the data look like that you are sending back? Maybe send a status code.

**Review by jvanboch — APPROVED** [review]
Looks good overall. Consider adding in status codes. line 52 of controller/index.js, not sure if that should be a strict equals or not (===).

File: server/index.js (hunk 2 above)
Comment (medinam6) [reply server/index.js:58]: It is sending an object with a message. example: 
{ "message": "Review was deleted successfully!" }

File: server/index.js (hunk 3 above)
Comment (medinam6) [reply server/index.js:72]: Same thing as delete: { "message": "Updated review with id 1203." }

File: controller/index.js (hunk 1 above)
Comment (medinam6) [reply controller/index.js]: Yes. thanks!

**Pull request merged**